lambdas/service/app.py

```diff
@@ -2204,7 +2207,58 @@ def cart_export_send_to_collection_api(event, context):
     }
 
 
-@app.route(drs.drs_http_object_path('{file_uuid}'), methods=['GET'], cors=True)
+drs_description = format_description('''
+    This is a partial implementation of the
+    [DRS 1.0.0 spec](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/).
+    Not all features are implemented. This endpoint acts as a DRS compatible
+    proxy for the underlying datastore.
+
+    Any errors encountered from the underlying datastore are forwarded on as
+    errors from this endpoint.
+''')
+
+
+@app.route(drs.drs_http_object_path('{file_uuid}'), methods=['GET'], cors=True, method_spec={
+    'summary': 'Get file DRS object',
+    'description': format_description('''
+        This endpoint returns object metadata, and a list of access methods that can
+        be used to fetch object bytes.
+    ''') + drs_description,
+    'parameters': [
+        params.path('file_uuid', str, description='UUID of the file to be fetched'),
+    ],
+    'responses': {
+        '200': {
+            'description': format_description('''
+                A DRS object is returned. Either `access_id` will be included or
+                else both `access_url` and `type` are included.
+
+                If an `access_url` is returned, `type` will be `https` and
+                result is a signed URL to the object.
+
+                If an `access_id` is returned, use it in the next request.
+            '''),
+            'content': {
```

Comment:
Noah and I added a schema helper for this `content` - `application/json` structure.

Reply:
I see this is still in https://github.com/DataBiosphere/azul/pull/1798 which isn't yet merged. Do you want me to chain on top of this branch?

Reply:
The helper is already on the `develop` branch. Search for `application/json`

Reply:
Thanks. I was only looking in `openapi/schema.py` before.